### PR TITLE
chore: update yarn lockfile

### PR DESCRIPTION
**Pull request by ChristianMurphy**
includes update to acorn parser

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "build": "tsc -b",
     "lint": "run-p lint:*",
     "lint:es": "cross-env PARSER_NO_WATCH=true eslint . --cache --ext js,md.mdx,ts,tsx -f friendly",
-    "lint:tsc": "tsc --noEmit",
+    "lint:tsc": "tsc",
```
Comment (ChristianMurphy) [inline package.json]: removed no emit due to
```
tsconfig.json:4:5 - error TS5053: Option 'noEmit' cannot be specified with option 'incremental'.

4     "noEmit": true
      ~~~~~~~~


Found 1 error.
```

File: tsconfig.json [hunk 2]
```diff
@@ -1,8 +1,5 @@
 {
   "extends": "./tsconfig.base.json",
-  "compilerOptions": {
-    "noEmit": true
```
Comment (ChristianMurphy) [inline tsconfig.json]: removed no emit due to
```
tsconfig.json:4:5 - error TS5053: Option 'noEmit' cannot be specified with option 'incremental'.

4     "noEmit": true
      ~~~~~~~~


Found 1 error.
```

File: package.json (hunk 1 above)
Comment (JounQin) [reply package.json]: We can use `tsc --incremental=false --noEmit`

Comment (ChristianMurphy) [reply package.json]: updated in tsconfig.json

**Pull request merged**